casper/src/test/rholang/TestSet.rho

```diff
@@ -1,6 +1,22 @@
 //scalapackage coop.rchain.rholang.unittest
 
 //requires ListOps
+contract @"delayForCh"(@n, notHangingCh, done) = {
+  for(@notHanging <- notHangingCh) {
+    if(notHanging == false) {
```

Comment:
Bit of a nitpick, but I think this function could look a little nicer if written as
```
contract @"delayForCh"(@n, notHangingCh, done) = {
  for(@notHanging <- notHangingCh) {
    if (n == 0 or notHanging) { done!(notHanging) }
    else { notHangingCh!(notHanging) | @"delayForCh"!(n - 1, *notHangingCh, *done) }
  }
}